schemas/IDLs/org.gel.models.metrics.avro/1.0.1/ReportedVsGenetic.avdl

```diff
@@ -0,0 +1,153 @@
+        union {null, double} ratioChrX;
+        union {null, double} ratioChrY;
+        union {null, double} avgCnvChrX; // should be here?
+        union {null, double} avgCnvChrY; // should be here?
+    }
+
+/**
+Mendelian inconsistencies
+========================================
+- fmendel reports a line per nuclear family
+- imendel reports a line per member of the family
+*/
+    record PerFamilyMendelErrors {
+        string fatherId;
+        string motherId;
+        union {null, double} numberOfOffspring;
+        union {null, double} numberOfMendelErrors;
+    }
+
+    record IndividualMendelErrors {
+        string sampleId;
+        union {null, double} numberOfMendelErrors;
+        union {null, double} rateMendelErrors;
+    }
+
+    record AggregatedIndividualMendelErrors {
+        string sampleId;
+        union {null, double} totalnumberOfMendelErrors;
+    }
+
+    record TotalNumberOfMendelErrors {
+        union {null, double} family;
+        union {null, array<AggregatedIndividualMendelErrors>} individual;
+    }
+
+    record LocusMendelSummary {
+        string sampleId;
+        string chr;
+        double code;
+        double numberOfErrors;
+    }
+
+    record MendelianInconsistencies {
+        union {null, array<PerFamilyMendelErrors>} perFamilyMendelErrors;
+        union {null, array<IndividualMendelErrors>} individualMendelErrors;
+        union {null, TotalNumberOfMendelErrors} totalNumberOfMendelErrors;
+        union {null, array<LocusMendelSummary>} locusMendelSummary;
+    }
+
+/** Relatedness
+========================================
+*/
+    record RelatednessPair {
+        string sampleId1;
+        string sampleId2;
+        union {null, double} ibd0;
+        union {null, double} ibd1;
+        union {null, double} ibd2;
+        union {null, double} piHat;
+    }
+
+    record FamilyRelatedness {
+        union {null, array<RelatednessPair>} relatedness;
+    }
+
+/** Evaluation
+========================================
+reportedVsGeneticSummary: familyPassesGvsRChecks, familyFailsACheck, familyMissingACheck
+*/
+    record CoverageBasedSexCheck {
+        string sampleId;
+        union {null, Sex} reportedPhenotypicSex;
+        union {null, KaryotypicSex} reportedKaryotypicSex;
+        union {null, KaryotypicSex} inferredSexKaryotype;
+        union {null, Query} sexQuery;
+    }
+
+    record MendelianInconsistenciesCheck {
+        string sampleId;
+        union {null, Query} mendelianInconsistenciesQuery;
+        union {null, string} comments;
+    }
+
+    record familyRelatednessCheck {
+        string sampleId1;
+        string sampleId2;
+        union {null, string} relationshipFromPedigree;
+        union {null, string} possibleRelationship;
+        union {null, Query} withinFamilyIBDQuery;
+    }
+
+    record Evaluation {
+        union {null, array<CoverageBasedSexCheck>} coverageBasedSexCheck;
+        union {null, array<MendelianInconsistenciesCheck>} coverageBasedSexCheck;
```

Comment:
this needs to be `mendelianInconsistenciesCheck`, not `coverageBasedSexCheck`

Reply:
You are completely right! Quick copy-pasting...